src/Mvc/test/ViewCompilationApps/ApplicationWithRazorSdkNeitherUsed/ApplicationWithRazorSdkNeitherUsed.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(StandardTestAppTfms)</TargetFrameworks>
+
+    <!--
+      Turning off precompilation on will also turn off Razor SDK.
+    -->
+    <MvcRazorCompileOnPublish>false</MvcRazorCompileOnPublish>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Reference Include="Microsoft.AspNetCore.Mvc" />
+    <Reference Include="Microsoft.AspNetCore.Server.Kestrel" />
+    <Reference Include="Microsoft.Extensions.Configuration.CommandLine" />
+    <Reference Include="Microsoft.Extensions.Logging.Console" />
+    <Reference Include="Microsoft.NET.Sdk.Razor" PrivateAssets="All" />
```

Comment:
```suggestion
```

You can remove this from all test asset projects. Microsoft.NET.Sdk.Razor cannot be resolved as a ProjectReference, and the targets it provides are bundled in Microsoft.NET.Sdk.Web anyways.